src/app/components/articleList/articleList.tsx

```diff
@@ -18,11 +19,12 @@ export default function ArticleList() {
         </Typography>
       </div>
       <CardList />
-      <div>
+      <div className={styles.buttonArea}>
```

Comment:
Do we need a wrapper? Can't we use the following button component without `fullWidth`?